package.json

```diff
@@ -43,6 +43,7 @@
     "gl-vec4": "^1.0.1",
     "glob": "^7.0.3",
     "imports-loader": "^0.6.5",
+    "istanbul-combine": "^0.3.0",
```

Comment:
Yes, which is needed to have the xml file in the format we send to cdash.  We send multiple lcov files to Codecov, which works without this, but cdash expects a single Cobertura file.